rabbitmq_status.py

```diff
@@ -93,8 +93,15 @@ def main():
 
     if r.ok:
         resp_json = r.json()  # Parse the JSON once
-        if any('channels' in connection and connection['channels'] > 1 for connection in resp_json):
-            status_err('Detected RabbitMQ connections with multiple channels. Please check RabbitMQ and all Openstack consumers')
+        # We routinely see connection['channels'] == 2, so we cannot alarm if
+        # connection['channels'] > 1.  Note that connection['channels'] does
+        # not appear to be the total number of channels using the connection,
+        # but rather the channel number which identifies it within a
+        # connection.  We'll assume if we have a channel w/ number 11 or
```

Comment:
@Apsu - thanks for the further clarification. Unless I'm misunderstanding you it sounds like you are contradicting the comment in this patch, specifically:

"Note that connection['channels'] does
+        # not appear to be the total number of channels using the connection,
+        # but rather the channel number which identifies it within a
+        # connection."

@mattt416 - had you considered making that value a metric and creating a maas alarm for it if the number is > 10?

I think for future reference it would be worth including a bit more detail in a comment/commit message for this error so that it is clear what 10 represents, why it was chosen.